backend/src/test/java/botobo/core/domain/workbook/WorkbookRepositoryTest.java

```diff
@@ -69,4 +76,42 @@ void isPublic() {
         // when, then
         assertThat(workbookRepository.existsByIdAndOpenedTrue(savedWorkbook.getId())).isTrue();
     }
+
+    @Test
+    @DisplayName("유저의 Workbook 최신순으로 조회 - 성공")
+    void findAllByUserId() {
+        // given
+        User user = User.builder()
+                .githubId(1L)
+                .userName("oz")
+                .profileUrl("github.io")
+                .role(Role.USER)
+                .build();
+        userRepository.save(user);
+
+        Workbook workbook1 = Workbook.builder()
+                .name("오즈의 Java")
+                .opened(true)
+                .deleted(false)
+                .user(user)
+                .build();
+
+        workbookRepository.save(workbook1);
+
+        Workbook workbook2 = Workbook.builder()
+                .name("오즈의 Spring")
+                .opened(true)
+                .deleted(false)
+                .user(user)
+                .build();
+
+        workbookRepository.save(workbook2);
+
+        // when
+        List<Workbook> workbooks = workbookRepository.findAllByUserId(user.getId());
+
+        //then
+        assertThat(workbooks.size()).isEqualTo(2);
+        assertThat(workbooks).containsExactly(workbook2, workbook1);
```

Comment:
```suggestion
        assertThat(workbooks).hasSize(2)
                .containsExactly(workbook2, workbook1);
```
이런 것도 되더라구요 😮
바꿀 필요는 없고 그냥 꿀팁!